plugins/jobs/src/js/data/JobModel.ts

```diff
@@ -186,7 +194,11 @@ export const resolvers = ({
           nodes: jobs
         }));
     },
-    job(_obj = {}, args: GeneralArgs, _context = {}): Observable<Job | null> {
+    job(
+      _parent = {},
+      args: GeneralArgs,
+      _context = {}
+    ): Observable<Job | null> {
```

Comment:
you added `!` in gql, so you can remove the `null` here :)

Reply:
Oh, I was not supposed to touch that type. Nice catch. I only wanted to change the _obj to _parent.